ic-os/setupos/config/config.ini

```diff
@@ -23,6 +13,26 @@ ipv6_prefix=2a00:fb01:400:44
 # Define the gateway address for your IPv6 network. Ensure it's within your network range:
 ipv6_gateway=2a00:fb01:400:44::1
 
+# ------------------------------
+# Node Reward Settings (required for some node operators)
+# ------------------------------
+# Specifies the type of node rewards that the node operator expects to receive.
+# The value must match a community-approved value in the NNS.
+#
+# Example command to get node operator details:
+# ic-admin --nns-url https://ic0.app get-node-operator ut325-qbq5v-fli2f-e2a5h-qapdd-fsuyv-xej2j-ogvux-i3fc2-5nj3a-2ae
+# Example output:
+# NodeOperator { node_operator_principal_id: ut325-qbq5v-fli2f-e2a5h-qapdd-fsuyv-xej2j-ogvux-i3fc2-5nj3a-2ae, node_allowance: 0, node_provider_principal_id: lq5ra-f4ibl-t7wpy-hennc-m4eb7-tnfxe-eorgd-onpsl-wervo-7chjj-6qe, dc_id: "bo1", rewardable_nodes: {"type1": 28}, ipv6: None }
+# ==> The community-approved value of node_reward_type for this operator is "type1"
+#
+# If not provided, the NNS will attempt to automatically assign a value based on the operator's reward configuration.
+# If the reward configuration in unique, the NNS will automatically assign the correct node_reward_type.
```

Comment:
```suggestion
# If the reward configuration is unique, the NNS will automatically assign the correct node_reward_type.
```
Is this suggestion right? I'm still confused by this sentence.

Reply:
Fair point. Let me massage it bit.